uikit-dz/uikit-dz/Cells/StoriesTableViewCell.swift

```diff
@@ -0,0 +1,297 @@
+// StoriesTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Ячейка для сторисов
+final class StoriesTableViewCell: UITableViewCell {
+    // перечисление для констант
+    enum Constant: String {
+        case story
+        case story2
+        case yourstory = "Ваша история"
+        case lavanda123
+        case plus
+        case identifier = "StoriesTableViewCell"
+    }
+
+    // MARK: - Public static properties
+
+    static let identifier = Constant.identifier.rawValue
+
+    // MARK: - Private ui elements
```

Comment:
```suggestion
    // MARK: - Private ui elements
```
марки почитай в вики не верно называешь и поправь везде